File: Address.cpp
#include "Address.h"



#include <iostream>
#include "Address.h"

Address::Address(string Street_Address,
	string City,
	string State,
	int Zip_Code) :
	street_address(Street_Address),
	city(City),
	state(State),
	zip_code(Zip_Code)
{}


void Address::Display() const
{
	cout.fill('0');
	cout << street_address << endl;
	cout << city << ", " << state << " ";
	cout.width(5);
	cout << zip_code << endl;
}

File: Address.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace VenueSort
{
    class Address
    {
        public string streetAddress { get; set; }
        public string city { get; set; }
        public string state { get; set; }
        public int zipCode { get; set; }

        //Constructor
        public Address(string Street, string City, string State, int Zip)
        {
            this.streetAddress = Street;
            this.city = City;
            this.state = State;
            this.zipCode = Zip;
        }

        public void Display()
        {
            Console.WriteLine(this.streetAddress);
            Console.WriteLine(this.city + ", " + this.state);
            Console.WriteLine(this.zipCode);
        }
    }
}
